spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -239,13 +237,10 @@ def assert_flavors
           end
 
         expect(flavor.ext_management_system).to eq @ems
-        expect(flavor.enabled).to                 eq true
-        expect(flavor.cpu_cores).to               eq nil
-        expect(flavor.description).to             eq nil
-        expect(flavor.root_disk_size).to          eq root_disk_size_by_flavor(flavor.name)
-        expect(flavor.swap_disk_size).to          eq flavor.name == "m1.ems_refresh_spec" ? 512.megabytes : 0
-        expect(flavor.ephemeral_disk_size).to     eq flavor.name == "m1.ems_refresh_spec" ? 1.gigabytes : 0
```

Comment:
@jvlcek btw. we should avoid such tests. It will not catch errors when there is 0 and there shouldn't be.

Seems like Grizzly environment doesn't support this, so I solved it by not comparing disk for Grizzly:
blacklisted_attributes += [:disk, :ephemeral, :swap] if environment_release_number < 4

but rest of the environments should have the expected value